File: my-folder/0058-length-of-last-word/solution.java
class Solution {
    public int lengthOfLastWord(String s) {
        String s1=s.trim();
        char[] chars=s1.toCharArray();
        int count=0;
        for(int i= chars.length-1;i>=0;i--){
            count++;
            if(chars[i]==' '){
                return count-1;
            }
        }
        return count;
    }
}

File: my-folder/0058-length-of-last-word/solution.py
class Solution:
    # Iteration
    def lengthOfLastWord(self, s: str) -> int:
        i = len(s)-1
        result = 0

        while s[i] == " ":
            i -= 1

        while i >= 0 and s[i] != " ":
            i -= 1
            result += 1

        return result

# Time -> O(n)
# Space -> O(1)
        
